File: Latihan b/tim Java/Tim.java
class Sepakbola
{ 
/*kelas yang digunakan untuk mengimplementasikan tim sepakbola*/
/*atribut tim sepakbola*/
		private String nama;		//namatim
		private String negara;		//negara asal tim
		private int tahun;			//tahun berdiri tim 
		private String pelatih;		//pelatih tim
		private String pemain_pos;	//beberapa pemain dan posisinya
		private String stadion;		//stadion utama
		private String liga;		//kompetisi tim 
	
		Sepakbola(){
		/*costructor*/
			tahun = 0;
		}
		
		Sepakbola(String a, String b, int c, String d, String e, String f, String g){
		/*costructor*/
			this.nama = a;
			this.negara = b;
			this.tahun = c;
			this.pelatih = d;
			this.pemain_pos = e;
			this.stadion = f;
			this.liga = g;
		}
	
	/*mengeset nilai nama*/
	public void setnama(String a)
	{
		this.nama = a;
	}
	/*mengeset nilai negara*/
	public void setnegara(String b)
	{
		this.negara = b;
	}
	/*mengeset nilai tahun*/
	public void settahun(int c)
	{
		this.tahun = c;
	}
	/*mengeset nilai pelatih*/
	public void setpelatih(String d)
	{
		this.pelatih = d;
	}
	/*mengeset nilai pemain*/
	public void setpemain_posisi(String e)
	{
		this.pemain_pos = e;
	}
	/*mengeset nilai stadion*/
	public void setstadion(String f)
	{
		this.stadion = f;
	}
	/*mengeset nilai liga*/
	public void setliga(String g)
	{
		this.liga = g;
	}
	
	/*mengembalikan nilai nama*/
	String getnama()
	{
		return this.nama;
	}
	/*mengembalikan nilai negara*/
	String getnegara()
	{
		return this.negara;
	}
	/*mengembalikan nilai tahun*/
	int gettahun()
	{
		return this.tahun;
	}
	/*mengembalikan nilai pelatih*/
	String getpelatih()
	{
		return this.pelatih;
	}
	/*mengembalikan nilai pemain*/
	String getpemain_posisi()
	{
		return this.pemain_pos;
	}
	/*mengembalikan nilai stadion*/
	String getstadion()
	{
		return this.stadion;
	}
	/*mengembalikan nilai liga*/
	String getliga()
	{
		return this.liga;
	}
	
	
}

File: Latihan b/Tim C++/tim.cpp
#include <iostream>
#include <string>


class Sepakbola
{
	/*kelas yang digunakan untuk mengimplementasikan tim sepakbola*/
	
	private:
	/*atribut tim sepakbola*/
		std::string nama;		//namatim
		std::string negara;		//negara asal tim
		int tahun;				//tahun berdiri tim 
		std::string pelatih;	//pelatih tim
		std::string pemain_pos;	//beberapa pemain dan posisinya
		std::string stadion;	//stadion utama
		std::string liga;		//kompetisi tim 
	
	public:
	/*costructor*/
		Sepakbola(){
			tahun = 0;
		}
		
		Sepakbola(std::string a, std::string b, int c, std::string d, std::string e, std::string f, std::string g){
			/*costructor*/
			nama = a;
			negara = b;
			tahun = c;
			pelatih = d;
			pemain_pos = e;
			stadion = f;
			liga = g;
		}
	
	/*mengeset nilai nama*/
	void setnama(std::string a)
	{
		nama = a;
	}
	/*mengeset nilai negara*/
	void setnegara(std::string b)
	{
		negara = b;
	}
	/*mengeset nilai tahun*/
	void settahun(int c)
	{
		tahun = c;
	}
	/*mengeset nilai pelatih*/
	void setpelatih(std::string d)
	{
		pelatih = d;
	}
	/*mengeset nilai pemain*/
	void setpemain_posisi(std::string e)
	{
		pemain_pos = e;
	}
	/*mengeset nilai stadion*/
	void setstadion(std::string f)
	{
		stadion = f;
	}
	/*mengeset nilai liga*/
	void setliga(std::string g)
	{
		liga = g;
	}
	
	/*mengembalikan nilai nama*/
	std::string getnama()
	{
		return nama;
	}
	/*mengembalikan nilai negara*/
	std::string getnegara()
	{
		return negara;
	}
	/*mengembalikan nilai tahun*/
	int gettahun()
	{
		return tahun;
	}
	/*mengembalikan nilai pelatih*/
	std::string getpelatih()
	{
		return pelatih;
	}
	/*mengembalikan nilai pemain*/
	std::string getpemain_posisi()
	{
		return pemain_pos;
	}
	/*mengembalikan nilai stadion*/
	std::string getstadion()
	{
		return stadion;
	}
	/*mengembalikan nilai liga*/
	std::string getliga()
	{
		return liga;
	}
	
	~Sepakbola(){
		/*destuktor*/
	}
	
};